mirai-core/src/commonMain/kotlin/contact/NormalMemberImpl.kt

```diff
@@ -129,6 +129,9 @@ internal class NormalMemberImpl constructor(
         check(this.id != bot.id) {
             "A bot can't mute itself."
         }
+        check(durationSeconds != 0) {
```

Comment:
```suggestion
        check(durationSeconds > 0) {
```

Reply:
fixed

Reply:
`require` is better, throws `IllegalArgumentException`

Reply:
:(